ccw.core/plugin.xml

```diff
@@ -3399,4 +3399,14 @@
          </bundle>
       </component>
    </extension>
+   <extension
+         point="org.eclipse.ui.themes">
+         <themeElementCategory label="%Presentation.label" id="ccw.editors.clojure.presentation"/>
+      <colorDefinition
+         id="ccw.editors.clojure.colors.repl.expressionBackground"
```

Comment:
This basically adds one or more colors to your software color registry. It is the color that was in CcwPlugin and I thought of refactoring it out. The framework resolves it using its id.